configs/default/MTKS-MATEKF405TE.config

```diff
@@ -0,0 +1,146 @@
+resource MOTOR 3 B15
+resource MOTOR 4 A08
+resource MOTOR 5 B11
+resource MOTOR 6 B10
+resource MOTOR 7 B03
+resource MOTOR 8 A15
+resource SERVO 1 B14
+resource SERVO 2 A06
+resource SERVO 3 B06
+resource PPM 1 A03
+resource LED_STRIP 1 B01
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 C10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_TX 11 A02
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 C11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 6 C07
+resource I2C_SDA 1 B07
+resource I2C_SCL 1 B08
+resource LED 1 A14
+resource LED 2 A13
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_MISO 1 B04
+resource SPI_MISO 2 C02
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 C03
+resource ADC_BATT 1 C04
+resource ADC_RSSI 1 B00
+resource ADC_CURR 1 C05
+resource ADC_EXT 1 C00
+resource SDCARD_CS 1 C01
+resource PINIO 1 A04
+resource PINIO 2 B05
+resource FLASH_CS 1 C13
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 C15
+resource GYRO_CS 1 C14
+
+# timer
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer B15 AF1
+# pin B15: TIM1 CH3N (AF1)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer B11 AF1
+# pin B11: TIM2 CH4 (AF1)
+timer B10 AF1
+# pin B10: TIM2 CH3 (AF1)
+timer B03 AF1
+# pin B03: TIM2 CH2 (AF1)
+timer A15 AF1
+# pin A15: TIM2 CH1 (AF1)
+timer B14 AF9
+# pin B14: TIM12 CH1 (AF9)
+timer A06 AF9
+# pin A06: TIM13 CH1 (AF9)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer B09 AF3
+# pin B09: TIM11 CH1 (AF3)
+timer A03 AF3
+# pin A03: TIM9 CH2 (AF3)
+timer A02 AF2
+# pin A02: TIM5 CH3 (AF2)
+
+
+# dma
+dma ADC 1 1
+# ADC 1: DMA2 Stream 4 Channel 0
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin C08 0
+# pin C08: DMA2 Stream 2 Channel 0
+dma pin B15 1
+# pin B15: DMA2 Stream 6 Channel 6
+dma pin A08 1
+# pin A08: DMA2 Stream 1 Channel 6
+dma pin B11 0
+# pin B11: DMA1 Stream 7 Channel 3
+dma pin B10 0
+# pin B10: DMA1 Stream 1 Channel 3
+dma pin B03 0
+# pin B03: DMA1 Stream 6 Channel 3
+dma pin A15 0
+# pin A15: DMA1 Stream 5 Channel 3
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin A02 0
+# pin A02: DMA1 Stream 0 Channel 6
+
+# feature
+feature OSD
+
+# serial
+serial 20 1 115200 57600 0 115200
+serial 0 0 115200 57600 0 115200
+serial 1 64 115200 57600 0 115200
+serial 2 1024 115200 57600 0 115200
+serial 3 1 115200 57600 0 115200
+serial 4 0 115200 57600 0 115200
+serial 5 0 115200 57600 0 115200
+serial 30 0 115200 57600 0 115200
+
+
+# master
+set current_meter = ADC
+set battery_meter = ADC
+set dshot_bitbang = ON
+set beeper_frequency = 2500
+set blackbox_device = SPIFLASH
+set max7456_spi_bus = 1
+set dashboard_i2c_bus = 0
+set pinio_box = 40,41,255,255
+set flash_spi_bus = 2
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set baro_i2c_address = 0
```

Comment:
Isn't this the default setting?

Reply:
This is generated from a `diff all bare` as per https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md so I I don't think so.